Gemfile

```diff
@@ -24,15 +24,10 @@ gem "warden"
 # Used for handling authorisation policies
 gem "pundit"
 
-# Build JSON APIs with ease [https://github.com/rails/jbuilder]
-gem "jbuilder"
-
 # Use Sentry (https://sentry.io/for/ruby/?platform=sentry.ruby.rails#)
 gem "sentry-rails"
 gem "sentry-ruby"
 
-gem "dotenv-rails", groups: %i[development test]
```

Comment:
Could it be that some people use this locally for development? (I discovered I was by accident this week, had a .env file I had forgotten about and didn't realise was used! 😮 )

Reply:
Oh I don't actually use it usually, I was wondering if anyone else did. I wanted to stash some env var values and threw them in an uncommitted .env file thinking I would just `source .env` if I needed them - little did I know that we used the dotenv gem and it tripped me up!

Reply:
I'm not using it

Reply:
Me neither. I guess if anyone _is_ using this they can use `source .env` for this as Jamie says?